### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by jacopotarantino** [jasmine-contrib/grunt-jasmine-node#66]
It took me a long time of reading the source code of this plugin and node-jasmine to try and understand what was wrong with my grunt config options. I have a pretty standard suite of tests(all written in CoffeeScript and couldn't get things to work to save my life. I finally got my configuration to work with the following(using load-grunt-config):

``` coffeescript
'use strict'

module.exports =
  projectRoot: 'spec'
  forceExit: true
  useCoffee: true
  extensions: 'coffee'
```

This config doesn't do exactly what I want but I was happy to get it to finally work at all. It would be very helpful if the documentation for this project were much more verbose so that I didn't have to do so much reading to get a simple config object to look right. I'd be happy to help with this documentation bit I need a bit of help getting started at least.

**Comment by brownknows** [comment]
Ditto.   Please document the options!!

**Comment by jackspaniel** [comment]
Yeah the problem seems to be the example on the readMe wraps the jasmine_node options in an object named 'options. Whereas the code expects the properties to be at the root of jasmine_node.

**Comment by jackspaniel** [comment]
FYI - I tried this project and it's working without having to move the jasmine_node options around. https://github.com/lennym/grunt-jasmine-node/blob/master/tasks/jasmine-node-task.js

The only thing I can't figure out is if I leave off the all: ['spec/'] property in the jasmine_node config, it fals to load any of my options. If I put all: anything, it works. Has to be some weird built in grunt thing.
